docs/adguard-vpn-browser-extension/solving-problems/logs.md

```diff
@@ -0,0 +1,64 @@
+---
+title: How to collect and send logs
+sidebar_position: 1
+---
+
+If you encounter a problem when using AdGuard VPN Browser Extension, you can inform us about it by sending application logs.
+
+## Collecting and sending standard logs
+
+By default, AdGuard VPN Browser Extension uses the Default logging level, that is, the basic collection of data about running processes of the app. To send these logs, follow the next steps:
+
+1. Open AdGuard VPN Browser Extension, if possible, repeat the actions that led to the error. Note the exact time when this error occurred.
+
+2. Open *Settings* by clicking the hamburger menu icon (☰) → *Support* → *Report a bug*.
+
+3. In the opened form, enter your email address for feedback and describe the error found, including the time when this error occurred. If you can't reproduce the problem, specify as accurately as possible when it last occurred.
+
+4. There is a check mark next to *Include the diagnostic report in the message*, which means that when you send a report, you also send logs.
+
+## Collecting and sending extended logs
```

Comment:
in the nearest version of the extension, we would add an export logs button

with the help of this button it would be possible to export logs if there are no possibilities to open the settings page, and without the necessity to open the background page of the browser.

<img width="471" alt="image" src="https://user-images.githubusercontent.com/17272769/218463285-82a6e99a-535c-4fb7-b2eb-1bffdb21f8e1.png">